test/storage/b_plus_tree_concurrent_test.cpp

```diff
@@ -333,4 +347,73 @@ TEST(BPlusTreeConcurrentTest, DISABLED_MixTest) {
   remove("test.log");
 }
 
+TEST(BPlusTreeConcurrentTest, DISABLED_MixTest2) {
+  // create KeyComparator and index schema
+  auto key_schema = ParseCreateStatement("a bigint");
+  GenericComparator<8> comparator(key_schema.get());
+
+  auto *disk_manager = new DiskManager("test.db");
```

Comment:
It would be a good idea to replace all DiskManager with InMemoryDiskManagerUnlimited (see project 1 benchmark for more information)

Reply:
do you mean DiskManagerUnlimitedMemory()?

Reply:
Yes 🤣